packages/tupaia-web/src/layout/ProjectCardList/ProjectCard.tsx

```diff
@@ -127,8 +128,11 @@ export const ProjectPendingLink = () => (
     Approval in progress
   </OutlineLink>
 );
-export const ProjectAllowedLink = ({ url }: LinkProps) => (
-  <BaseLink to={url}>View project</BaseLink>
+
+export const ProjectAllowedLink = ({ url, isLandingPage }: LinkProps) => (
+  <BaseLink to={url} target={isLandingPage}>
```

Comment:
So with this one, this is on the right track. But you can even make `isLandingPage` just a `boolean` and when it is true, `target` will be `_blank`, otherwise it is `_self`. In this case, passing in a prop for `target` works but the naming convention isn't quite right because it implies a boolean but is a string. Personally I'd prefer the approach of just passing in `true/false` for `isLandingPage` and the `ProjectAllowedLink` then handling the `target` from that.